tests/unit/test_db_connector.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from truck_delivery_yess.db_connector import DBConnector
+
+
+def test_save_mock(mocker):
+    mocker.patch("truck_delivery_yess.db_connector.DBConnector.save").return_value = True
```

Comment:
as you mentioned before, we should mock Redis methods instead of the method we are tryign to tests, so for that you can use:
```   
    mocker.patch("redis.Redis.set").return_value = True
    mocker.patch("redis.Redis.get").return_value = json.dumps(object_to_save)
` ``
same for the other test on this file